### DISCUSSION TITLE
PowerShell

### DISCUSSION THREAD

**Discussion by SirParser** [mini-software/MiniExcel#345] [Q&A] [answered]
Anyone knows how to use this command in PowerShell:
	MiniExcel.SaveAs(path, value, overwriteFile: true)
(the "overwriteFile: true" part) ?

**Comment by shps951023** [comment, marked as answer]
@HiTekBot 
you can try this
```C#
MiniExcel.SaveAs(path, value,  true, "Sheet1", ExcelType.UNKNOWN, null,false)
```

**Reply by SirParser** [reply]
It looks like this is available from version 1.25.0:
_or, since V1.25.0, SaveAs support overwriteFile parameter for enable/unable overwriting exist file_
... but the latest available version is 1.24.3
Any idea when 1.25.0 will be released?

**Reply by shps951023** [reply]
@HiTekBot  60% this weekend

**Reply by SirParser** [reply]
Thanks, it works now
